classes/submission/maps/Schema.php

```diff
@@ -165,4 +175,66 @@ protected function getAvailableEditorialDecisions(int $stageId, Submission $subm
 
         return $decisionTypes;
     }
+
+    /**
+     * Get issues associated with submissions. Results are keyed by submission ID.
+     *
+     * @return Enumerable<int, Issue[]>
+     */
+    protected function getSubmissionsIssues(Enumerable $submissions): Enumerable
+    {
+        $submissionIds = $submissions->map(fn (Submission $submission) => $submission->getId())->all();
+        $publications = Repo::publication()->getCollector()->filterBySubmissionIds($submissionIds)->getMany();
+        $issueIds = $publications->map(fn (Publication $publication) => $publication->getData('issueId'))
+            ->unique()
+            ->all();
```

Comment:
I guess the `Submission` object is supposed to have data in the `getData('publications')`, which will be a list of publications, then the `$submissionIds` can be removed and the `$publications` can be optimized.